templates/ecoscore_details.tt.html

```diff
@@ -43,19 +56,34 @@ <h4>Origine des ingrédients</h4>
 		<strong>Bonus / malus - Politique environnementale : [% round(adjustments.origins_of_ingredients.epi_value) %]</strong><br>
```

Comment:
"Bonus / malus - Politique environnementale : -5" is hard to read. I had to understand that the first "-" is not a "minus" but just a "connection" sign. I would write a ">" instead or event nothing:
=> "Bonus / malus > Politique environnementale : -5"